### DISCUSSION TITLE
When will tigramite 5.x be uploaded to Anaconda?

### DISCUSSION THREAD

**Discussion by dharaik** [jakobrunge/tigramite#280] [General]
It would be good to have tigramite 5.x on anaconda. Package versions in current tigramite create conflics in my native python environment. Installing using a package manager capable of resolving version conflicts like 'conda' would be a great help. I have to maintain different environments (one for tigarmite and one to process the results further using packages like cartopy, shapely, pymannkendall etc) due to numpy version mismatches.

**Comment by sagarsimha** [comment]
Thanks for the feedback. We are working on this.
